include/tcpdf_static.php

```diff
@@ -1859,7 +1859,7 @@ public static function url_exists($url) {
 	public static function encodeUrlQuery($url) {
 		$urlData = parse_url($url);
 		if (isset($urlData['query']) && $urlData['query']) {
-			$urlQueryData = array();
+			$urlQueryData = [];
```

Comment:
You should revert this change to keep maximal php versions coverage

Reply:
I saw that in several other parts of the code that are already short array syntax is used, so i've changed it there too.

Reply:
Ah, nevermind. I was wrong, i'll revert.